scripts/analysis.py

```diff
@@ -56,188 +25,85 @@ def get_agent_ids(cursor, facility):
     cursor: cursor
         cursor for sqlite3
     facility: str
-        name of facility type
+        name of facility type as described in spec
```

Comment:
You are correct, the agententry table includes a column called 'spec', that describes the agent's archetype specification.
I will rename the parameter as something more meaningful.